src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate(callback, item) {
+      this.result = callback(this.result, item);
```

Comment:
Ok, but what is `item`? I guess `number` will be more understandable in this case (it's a number that we want to use for calculation)